tests/functional/test_install.py

```diff
@@ -270,6 +270,20 @@ def test_install_from_local_directory(script, data):
     assert egg_info_folder in result.files_created, str(result)
 
 
+def test_install_from_local_directory_with_forward_slash(script, data):
+    """
+    Test installing from a local directory.
+    """
+    to_install = data.packages.join("FSPkg")
+    result = script.pip('install', to_install, expect_error=False)
+    fspkg_folder = "/".join([script.site_packages, 'fspkg'])
+    egg_info_folder = "/".join([
+        script.site_packages, 'FSPkg-0.1.dev0-py%s.egg-info' % pyversion
+    ])
+    assert fspkg_folder in result.files_created, str(result.stdout)
+    assert egg_info_folder in result.files_created, str(result)
+
```

Comment:
Not a problem - it's good you added a test at all, it's easy with something like this to think it's simple enough to not need a test (a mistake I make all too often 😄)